files/zh-cn/learn/css/first_steps/getting_started/index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: 让我们开始 CSS 的学习之旅
+slug: Learn/CSS/First_steps/Getting_started
+tags:
+  - CSS
+  - 元素
+  - 初学者
+  - 学习
+  - 类
+  - 选择器
+translation_of: Learn/CSS/First_steps/Getting_started
+original_slug: Learn/CSS/First_steps/开始
+---
+{{LearnSidebar}}
+
+{{PreviousMenuNext("Learn/CSS/First_steps/What_is_CSS", "Learn/CSS/First_steps/How_CSS_is_structured", "Learn/CSS/First_steps")}}
+
+在这篇文章中，我们将会拿一个简单的 HTML 文档做例子，并且在里面使用 CSS 样式，期待你能在此过程中学会更多有关 CSS 的实战性知识。
+
+#### 前置知识
+
+在开始本单元之前，您应该：
+
+- 基本熟悉计算机操作。
+- 基本工作环境的设置（详见[安装基本软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software)），基本的文件操作，详见[处理文件](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)。
+- 熟悉 [HTML 概述](/zh-CN/docs/Learn/HTML/Introduction_to_HTML) 章节中提到的基本 HTML 知识。
+
+#### 目标
+
+- 理解 HTML 文档链接 CSS 文档的几个基本套路，
+- 并能运用所学的 CSS，做些文字上的格式化操作。
+
+## 先从 HTML 开始吧
+
+我们先以 HTML 文档展开叙述。如果想在自己电脑试一试，可以复制下面的代码。在你的电脑上，将代码以文件名： `index.html` 的形式保存下来。
+
+```html
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>开始学习 CSS</title>
+</head>
+
+<body>
+
+    <h1>我是一级标题</h1>
+
+    <p>这是一个段落文本。在文本中有一个 <span>span element</span>
+并且还有一个 <a href="http://example.com">链接</a>.</p>
+
+    <p>这是第二段。包含了一个 <em>强调</em> 元素.</p>
+
+    <ul>
+        <li>项目 1</li>
+        <li>项目 2</li>
+        <li>项目 <em>三</em></li>
+    </ul>
+
+</body>
+
+</html>
+```
+
+> **注意：** 假设你电脑操作环境不方便创建文件运行代码，别担心，可以用我们下方给出的在线实时代码编辑器。</strong></p></div>
```

Comment:
the prefix for three note cards: <https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN#%E5%A4%87%E6%B3%A8%E3%80%81%E8%AD%A6%E5%91%8A%E5%92%8C%E6%A0%87%E6%B3%A8>

```suggestion
> **备注：** 假设你电脑操作环境不方便创建文件运行代码，别担心，可以用我们下方给出的在线实时代码编辑器。
```